rotkehlchen/db/upgrades/v31_v32.py

```diff
@@ -155,6 +155,13 @@ def _update_history_entries_from_kraken(cursor: 'Cursor') -> None:
         )
 
 
+def _update_settings_name_for_selected_binance_markets(cursor: 'Cursor') -> None:
+    cursor.execute("""
+    UPDATE user_credentials_mappings SET setting_name = "binance_selected_trade_pairs"
```

Comment:
This is kind of a nitpick, but why did you resolve my previous comment here: https://github.com/rotki/rotki/pull/4209#discussion_r853795192 without using the constant you defined in db/constants.py as I asked?

Reply:
Oops! That was clumsy from my end. Sorry about that.